reporter/src/main/kotlin/reporters/CycloneDxReporter.kt

```diff
@@ -100,7 +100,7 @@ class CycloneDxReporter : Reporter {
     ): List<File> {
         val outputFiles = mutableListOf<File>()
         val projects = input.ortResult.getProjects(omitExcluded = true)
-        val createSingleBom = options["single.bom"].isTrue()
+        val createSingleBom = !options["single.bom"].isFalse()
```

Comment:
The header comment of the class probably needs to be adapted to reflect this change of the default behaviour.

Reply:
Right, forgot about that! Fixed.